### ISSUE TITLE
Mac: TypeScript language service fails

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#1094]
I think since the update to 1.7 the language service stops working for me quickly. The ts server is still running but no smartness comes from the editor. Is there a way to enable diagnostics to find out what is happening?

**Comment by dbaeumer** [comment]
- Set the env variable TSS_DEBUG to an available port number.
- start VSCode

I do have tracing options in the TS extension but they can't be activated anymore since the Debug Console doesn't understand require. I will look into making this available via an env as well.

**Comment by dbaeumer** [comment]
@bpasero when you see it, please drag me in so that we can debug it.

**Comment by dbaeumer** [comment]
@bpasero I added support to enable tracing via env as well (see #1100). With a latest build set the env TSS_TRACE to true and start code. Open the developer tools and you should see output like:

```
[Plugin Host] TypeScript Service: sending request open(0). Response expected: no. Current queue length: 0
[Plugin Host] TypeScript Service: sending request geterr(1). Response expected: no. Current queue length: 0
```

**Comment by bpasero** [comment]
Thanks, lets check tomorrow.

**Comment by egamma** [comment]
will be addressed by #1198
